petab/problem.py

```diff
@@ -88,7 +88,8 @@ def __setstate__(self, state):
     @staticmethod
     def from_files(
             sbml_file: Union[str, Path, None] = None,
-            condition_file: Union[str, Path, None] = None,
+            condition_file:
+            Union[str, Path, None, Iterable[Union[str, Path]]] = None,
```

Comment:
Unrelated, but `parameter_file` can also be `Iterable[Union[str, Path]]` here, but is `List[str]` in `get_parameter_df` [1]. I guess it can be changed to `Iterable[Union[str, Path]]` at [1] too.

[1] https://github.com/PEtab-dev/libpetab-python/blob/c457c05172234ff0998eb3054b495f472fd9d0c3/petab/parameters.py#L31=

Reply:
Will address separately